### PR TITLE
Finalize habitat content DAH-1065

### PR DESCRIPTION
**Pull request by christine-sfg**
DAH-1065

Update habitat listing links and content based on the handoff figma

### CODE REVIEW

File: app/assets/javascripts/short-form/directives/communityScreeningHeader.js.coffee [hunk 1]
```diff
@@ -39,7 +39,7 @@ angular.module('dahlia.directives')
       when 'Habitat for Humanity'
         scope.title = $translate.instant('a2_community_screening.habitat.you_or_anyone')
         # FIXME: Update the habitat url
```
Comment (akegan) [inline app/assets/javascripts/short-form/directives/communityScreeningHeader.js.coffee:41]:
```suggestion
```

**Review by akegan — CHANGES_REQUESTED** [review]
LGTM with one minor change needed.

We need to update `listings.habitat.income_range.p1` to have 82%-109.8% as the AMI range.

**Comment by christine-sfg** [comment]
> LGTM with one minor change needed.
> 
> We need to update `listings.habitat.income_range.p1` to have 82%-109.8% as the AMI range.

I just noticed the figma has "between 89% and 109.8% AMI". Please confirm the correct values.

**Comment by akegan** [comment]
82 is the correct value. 89 was a typo. Msol [confirmed in slack too](https://sfdigitalservices.slack.com/archives/CCV0019TN/p1643846189928729)

File: app/assets/json/translations/locale-en.json [hunk 2]
```diff
@@ -943,7 +943,7 @@
                 "income_range": {
                     "income_range_plural": "{{number}} people household: {{minIncome}} to {{maxIncome}} per year",
                     "income_range_singular": "{{number}} person household: {{minIncome}} to {{maxIncome}} per year",
-                    "p1": "To be eligible for a Habitat for Humanity Greater San Franscisco home in this listing, your household income needs to be between 69% and 109% AMI as listed below.",
+                    "p1": "To be eligible for a Habitat for Humanity Greater San Francisco home in this listing, your household income needs to be between 82% and 109.8% AMI as listed below.",
```
Comment (akegan) [inline app/assets/json/translations/locale-en.json:946]: nice catch on the typo!

**Review by akegan — APPROVED** [review]
LGTM now. Just kicked off tests again 🤞

**Pull request merged**